src/content/docs/en/pages/devtools/api-graphql/features/metrics-fields.mdx

```diff
@@ -237,6 +237,40 @@ When a field is a result of any kind of calculation, such as a sum, they’re co
 
 ---
 
+## botManagerMetrics
+
+:::note
+To retrieve this data, you must be subscribed to Azion Bot Manager. Contact the [Sales team](https://www.azion.com/en/contact-sales/) for more details on the subscription.
+:::
+
+| Field | Description |
+|----------|----------|
+| `action` | Action performed by Azion Bot Manager for accesses identified as Bots. Example: `deny`  |
+| `botCategory` | Bot category identified in the request. Example: `scraping`, `crawling`, `brute-force` |
+| `botMode` | Bot protection mode used in the request. Example: `Web` |
+| `challengeSolved ` | Total number of bots that either passed or failed the CAPTCHA challenge. This is a boolean, where `true` indicates **Solved** and `false` indicates **Not Solved**. |
```

Comment:
Na verdade não é exatamente "total number", talvez algo como: 
```suggestion
| `challengeSolved ` | Flag that indicates if the bot either passed or failed the CAPTCHA challenge. This is a boolean, where `true` indicates **Solved** and `false` indicates **Not Solved**. |
```
E o valor em sí é 0 e 1 (0 = not solved e 1 = solved)